app/controllers/catalog_controller.rb

```diff
@@ -191,7 +191,7 @@ def exclude_unwanted_records(solr_parameters,user_parameters)
     
     config.add_search_field ('Keyword (Title, Name/Creator, Topic, Notes etc.)') do |field|
 	  #field.solr_parameters = { :'qf' => 'all_fields_tesim' }
-	  field.solr_parameters = { :'qf' => 'title_tesim^99 name_tesim^20 subject_tesim^10 scopeContentNote_tesim^6 all_fields_tesim fulltext_tesim id' }
+	  field.solr_parameters = { :'qf' => 'title_tesim^99 name_tesim^20 topic_tesim^10 scopeContentNote_tesim^6 all_fields_tesim fulltext_tesim id' }
```

Comment:
Will it change the display order in the search result page for other subject types like genre, geographic etc?